.github/workflows/continuous-integration.yml

```diff
@@ -31,7 +31,7 @@ jobs:
           restore-keys: |
             ${{ runner.os }}-node-
 
-      - run: npm install
```

Comment:
Das sollten wir auch in unseren docker containern so machen, aber dann im nächsten PR.
Issue hier: https://github.com/ecamp/ecamp3/issues/775